### DISCUSSION TITLE
No dp file after building from source using C++ interface

### DISCUSSION THREAD

**Discussion by Cheng-JW** [deepmodeling/deepmd-kit#3084] [Q&A]
I have built the software from source using C++ interface following the instruction. But there's no "dp" file in $deepmd_root/bin, where I installed it. There's only "dp_gmx_patch" and "dp_ipi". Is there any problem and where can I find the dp file?
deepmd-kit version: 2.2.7
tensorflow version: 2.15.0-post1
cmake command: cmake -DUSE_CUDA_TOOLKIT=TRUE -DUSE_TF_PYTHON_LIBS=TRUE -DCMAKE_INSTALL_PREFIX=$deepmd_root -DCMAKE_CXX_COMPILER=/gpfs/share/software/gcc/10.4.0/bin/g++ -DCMAKE_C_COMPILER=/gpfs/share/software/gcc/10.4.0/bin/gcc ..

**Comment by njzjz** [comment]
`dp` is installed by the Python interface
